libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -73,8 +74,39 @@ bool ControlFlowBuilder::visit(BinaryOperation const& _operation)
 			return false;
 		}
 		default:
-			return ASTConstVisitor::visit(_operation);
+		{
+			bool result = ASTConstVisitor::visit(_operation);
```

Comment:
I think we can just call `visitNode` here and ignore its return value, like in all the other cases (see above)

Reply:
Sure, I will use visitNode here. But, visitNode call does not lead to children visits so return value must be `true`. It was `true` before, since` STConstVisitor :: visit` always returns true. In the above code section, `appendControlFlow` pass visitor to children, therefore the function returns false.

Reply:
Sure, I will use `visitNode`. Also, as mentioned on the team channel, I will return `true` from the function so children nodes are also visited.